src/core.js

```diff
@@ -1,8 +1,25 @@
+/*!
+ * QUnit @VERSION
```

Comment:
Replacing "@VERSION" was done by the concat task, via `process` option. Does browserify handle that?